.github/workflows/integration-tests-against-emulator.yaml

```diff
@@ -24,9 +24,17 @@ jobs:
           python-version: 3.8
       - name: Install nox
         run: python -m pip install nox
-      - name: Run system tests
+      - name: Run system tests (without extended tracing)
         run: nox -s system
         env:
           SPANNER_EMULATOR_HOST: localhost:9010
           GOOGLE_CLOUD_PROJECT: emulator-test-project
           GOOGLE_CLOUD_TESTS_CREATE_SPANNER_INSTANCE: true
+          EXTENDED_TRACING_ENABLED: false
+      - name: Run system tests (with extended tracing)
```

Comment:
Why do we need a seperate job for extended tracing? What are we validating with the help of this job?

Reply:
It's to ensure that extended tracing works as expected because there is no other mechanism to test for it.

Reply:
EXTENDED_TRACING_ENABLED will add sql statements in span attributes right?
I don't think we need to run a job explicitly for that if we don't have any tests specifically to validate this behavior in system tests.